contrib/builder/rpm/generate.sh

```diff
@@ -96,5 +109,10 @@ for version in "${versions[@]}"; do
 
 	echo 'ENV AUTO_GOPATH 1' >> "$version/Dockerfile"
 
-	echo 'ENV DOCKER_BUILDTAGS selinux' >> "$version/Dockerfile"
+	DOCKER_BUILDTAGS="selinux"
+	if [[ "$distro" == "amazon" ]]; then
+		DOCKER_BUILDTAGS="$DOCKER_BUILDTAGS !journald"
```

Comment:
The main purpose of matching was to match based on what version of RPM and `rpmbuild` we have, which is closest to CentOS 7.

I'll look into changing the build image, although I'd strongly prefer the RPMs get built with a newer version of `rpmbuild`.